cmd/utils/flags.go

```diff
@@ -111,6 +112,12 @@ func printHelp(out io.Writer, templ string, data interface{}) {
 // are the same for all commands.
 
 var (
+	// Genesis settings
+	GenesisFlag = cli.StringFlag{
+		Name:  "genesis",
+		Usage: "Path to genesis JSON file, ignore genesis block from database",
+		Value: "./genesis.json",
```

Comment:
I don't think it should provide a default path for genesis, since current network already set genesis config (including  hard forks) in code.

Reply:
[4fc5bca](https://github.com/dogechain-lab/dbsc/pull/9/commits/4fc5bca656e7707666dc38d1c22dc5501c03a9d7) remove it.